src/test/common/utils/interest.ts

```diff
@@ -38,23 +38,23 @@ describe('getInterestDetails', () => {
     expect(await getInterestDetails(claim)).to.be.eq(undefined)
   })
 
-  it('should return 1 for number of days when interest date starts from today', async () => {
+  it('should return 0 for number of days when interest date starts from today', async () => {
     claimStoreServiceMock.mockCalculateInterestRate(5)
 
     const today = MomentFactory.currentDate()
     const claim: Claim = new Claim().deserialize({ ...sampleClaimObj, createdAt: today })
```

Comment:
Shouldn't we set `issuedOn` for the test case?

Reply:
oops! I changed the other one but forgot this one. Will change it. thanks!